backend/src/BIE.DataPipeline/Program.cs

```diff
@@ -94,34 +92,36 @@
 {
     var line = "";
     var notEof = importer.ReadLine(out line);
+    Console.WriteLine("Inserting the data into the database...");
 
-    Console.WriteLine("Inserting into Database");
-
+    // Read all lines
     var count = 0;
     while (notEof)
     {
         dbHelper.InsertData(line);
         notEof = importer.ReadLine(out line);
         count++;
-        Console.Write($"\rLines: {count}");
+        if (count % 10000 == 0)
```

Comment:
i wonder if this will improve the performance, or if it would be best to leave out the writes anyways.

Reply:
Yes, I did it just to improve the logging, because before it was impossible to see anything with the spam of "Lines X". The print here is just to inform us that the pipeline is still working, but we could delete it anyways if not needed.

Reply:
EDIT: Deleted the print.